tools/perfscale-audit/metric-client/metric-client.go

```diff
@@ -49,9 +49,22 @@ const (
 //
 //	at all times.
 const (
-	vmiPhaseCount = `sum by (phase) (kubevirt_vmi_phase_count{})`
+	vmiPhaseCount                 = `sum by (phase) (kubevirt_vmi_phase_count{})`
+	avgVirtAPICPUUsage            = `avg(sum(rate(container_cpu_usage_seconds_total{namespace="kubevirt",pod=~"virt-api.*", container!="",container!="POD"}[%ds]))by (pod))`
+	avgVirtAPIMemUsageInMB        = `avg(avg_over_time(container_memory_working_set_bytes{pod=~"virt-api.*", container!="POD", container!=""}[%ds]))/1024/1024`
```

Comment:
@Sreeja1725 can we use the `container_memory_rss` metric instead of working_set. Downstream we have learnt that rss is a more accurate measure of active memory used by the container, the working set includes some noise like inactive kernel pages etc that needs to be flushed

Reply:
Updated the metric